core/src/test/java/kafka/testkit/TestKitNodes.java

```diff
@@ -53,6 +57,8 @@ public static class Builder {
         private Map<Integer, Map<String, String>> perServerProperties = Collections.emptyMap();
         private BootstrapMetadata bootstrapMetadata = BootstrapMetadata.
             fromVersion(MetadataVersion.latestTesting(), "testkit");
+        private ListenerName listenerName = BROKER_LISTENER_NAME;
+        private SecurityProtocol securityProtocol = BROKER_SECURITY_PROTOCOL;
```

Comment:
we don't need this default value as in IT the default value is defined by `ClusterTest`. We can add null check for `securityProtocol` instead

Reply:
Not only ClusterTest use TestKitNode, there are many places like BrokerMetadataPublisherTest, KRaftClusterTest, ZkMigrationIntegrationTest directly TestKitNodes. That means we still need this default value.